### ISSUE TITLE
[BUG] Pure MM config walkthrough stops in order amount

### ISSUE THREAD

**Issue by dennisocana** [CoinAlpha/hummingbot#1451]
**Describe the bug**

Running into an error when trying to complete the configuration walkthrough.

```
2020-02-24 14:27:14,585 - 10 - hummingbot.client.hummingbot_application - ERROR - Unknown error while writing config. 'NoneType' object is not iterable
Traceback (most recent call last):
  File "/hummingbot/client/command/config_command.py", line 375, in _config_loop
    await self._inner_config_loop(keys)
  File "/hummingbot/client/command/config_command.py", line 363, in _inner_config_loop
    await self._inner_config_loop(self._get_empty_configs())
  File "/hummingbot/client/command/config_command.py", line 358, in _inner_config_loop
    value = await self.prompt_single_variable(cv, requirement_overwrite=False)
  File "/hummingbot/client/command/config_command.py", line 312, in prompt_single_variable
    val = await self.app.prompt(prompt=cvar.prompt, is_password=cvar.is_secure)
  File "/hummingbot/client/config/config_var.py", line 34, in prompt
    return self._prompt()
  File "/hummingbot/strategy/pure_market_making/pure_market_making_config_map.py", line 53, in order_amount_prompt
    min_amount = minimum_order_amount(trading_pair)
  File "/hummingbot/client/config/config_helpers.py", line 357, in minimum_order_amount
    default_quote_asset, default_amount = default_min_quote(quote_asset)
  File "/hummingbot/client/config/config_helpers.py", line 348, in default_min_quote
    global_quote_amount = [[b, m] for b, m in global_config_map.get("min_quote_order_amount").value if b == quote_asset]
TypeError: 'NoneType' object is not iterable
```

**Steps To Reproduce**

1. Create a new Hummingbot instance on the most recent docker development image
2. Start Hummingbot
3. Create a new pure market making strategy config file or import an existing one
4. Attempt to complete the configuration by answering all prompts during walkthrough

**Screenshots**

<img width="947" alt="putty_CiqEd6AWwX" src="https://user-images.githubusercontent.com/50150287/75161053-1cc99b80-5756-11ea-9bbb-8ef9715843f1.png">

**Release version**

dev-0.24.0

```
"branch": "development",
"commit": "22b0651d424e6b6938a15031d2b10e7950b8608f",
"date": "2020-02-22T00:33:20Z",
```

**Attachments**

[logs_hummingbot.log](https://github.com/CoinAlpha/hummingbot/files/4245054/logs_hummingbot.log)
[screen_recording.zip](https://github.com/CoinAlpha/hummingbot/files/4245034/puremm_config.zip)

**Comment by dennisocana** [comment]
Created a GitHub issue for users who are currently using Docker and running in the development version. Confirmed this is fixed in development branch via source-install.

**Comment by MishaelTereso** [comment]
Tested this bug using Docker running the most recent docker development image.

Steps Taken:
1. Created a new Hummingbot instance
2. Execute the configuration walkthrough
3. Created a new pure market making strategy config file
4. Attempted to complete the configuration and was able to answer all prompts during walkthrough

![image](https://user-images.githubusercontent.com/56957119/75232759-b9da1200-57f2-11ea-8b5e-ba4133871c49.png)

**Comment by dennisocana** [comment]
"branch": "development",
"commit": "4f6c72cb52d8a3193f1d16f20f973220d6cf946c",
"date": "2020-02-25T02:44:12Z",

**Comment by rennel-tabing** [comment]
I am able to reproduce this using a docker image.

**Comment by Clint-G13** [comment]
Confirm fix for this issue, since PR is not yet merged. I set values for min_quote_order_amount base on the PR.
![Capture](https://user-images.githubusercontent.com/57189990/75271612-72c24000-5837-11ea-8fce-1e715e0d0b7e.PNG)

```
"branch": "development",
"commit": "4f6c72cb52d8a3193f1d16f20f973220d6cf946c",
"date": "2020-02-25T02:44:12Z",
```

**Comment by Hashira-Lars** [comment]
Tested with newly installed .exe hummingbot (master branch) = [confirm fix] merged with pr #1456 

![image](https://user-images.githubusercontent.com/55869559/75298959-c69c4b80-586e-11ea-90ad-11a76ad97579.png)
